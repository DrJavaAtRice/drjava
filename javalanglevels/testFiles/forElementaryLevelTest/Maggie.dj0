class Maggie extends Object { int i, j;
  int foo() {
    System.out.println("Hello World.");
    return i;
  }
}