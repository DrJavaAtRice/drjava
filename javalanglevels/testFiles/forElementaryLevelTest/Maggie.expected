// Language Level Converter line number map: dj*->java. Entries: 6
//     1->3         2->4         3->5         4->6         5->7         6->8    
class Maggie extends Object { private final int i, j;
  public int foo() {
    System.out.println("Hello World.");
    return i;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public Maggie(int i, int j) {
    super();
    this.i = i;
    this.j = j;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int i() {
    return i;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int j() {
    return j;
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        i() + ", " + 
        j() + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      Maggie cast = ((Maggie) o);
      return (i() == cast.i()) && 
          (j() == cast.j());
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode() ^ 
        i() ^ 
        j();
  }
}
